runelite-client/src/main/java/net/runelite/client/plugins/customcursor/CustomCursorPlugin.java

```diff
@@ -85,22 +89,24 @@ private void updateCursor()
 
 		if (selectedCursor == CustomCursor.CUSTOM_IMAGE)
 		{
-			// Try to load the user cursor from ~/.runelite/cursor.png
-			File file = new File(RuneLite.RUNELITE_DIR, "cursor.png");
```

Comment:
I don't have strong feelings either way, but it seems like trying to load the custom cursor each time the cursor is updated is a good approach, rather than locking it in upon plugin initialization.

Reply:
This is not loading cursor, just creating File instance that is just reference to path